logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/auth/LogoutUseCaseTest.kt

```diff
@@ -127,6 +128,58 @@ class LogoutUseCaseTest {
             .wasInvoked(exactly = once)
     }
 
+    @Test
+    fun givenRemovedClientOrDeletedAccount_whenLoggingOutWithWipeOnDeviceRemovalEnabled_thenExecuteAllRequiredActions() = runTest {
```

Comment:
Previous test which covers the situation when we have wipe disabled tests both cases: 
```for (reason in listOf(LogoutReason.REMOVED_CLIENT, LogoutReason.DELETED_ACCOUNT)) { ...```
I think this one should do the same, better be sure that it's valid for both reasons if this test is supposed to cover both 😄

Reply:
Yes, actually needs to duplicate this function and call with `DELETED_ACCOUNT`

Reply:
done